<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Source_Language">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Source_Text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>691</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Translate_Button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Target_Language">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Target_Text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>691</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
